### PR TITLE
Remove jQuery dependency -https://issues.apache.org/jira/browse/SLING-9981

### PR DESCRIPTION
**Pull request by SBPrakash**
added javascript to make ajax call instead of jquery ajax as this requires jquery to be loaded on top of the page

### CODE REVIEW

File: src/main/resources/generators/javascript.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<div id="${uniqueId}"></div>
+<script type="text/javascript">
+    if (window.XMLHttpRequest) {
+        var xhr = new XMLHttpRequest();
+        xhr.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
+        xhr.timeout = 10000;
```
Comment (klcodanr) [inline src/main/resources/generators/javascript.html:24]: @SBPrakash - any particular reason to set the header and timeout?

File: src/main/java/org/apache/sling/dynamicinclude/generator/types/JsiGenerator.java [hunk 2]
```diff
@@ -39,7 +39,7 @@
  */
 @Component
 public class JsiGenerator implements IncludeGenerator {
-    private static final String TEMPLATE_FILENAME = "generators/jquery.html";
+    private static final String TEMPLATE_FILENAME = "generators/javascript.html";
```
Comment (klcodanr) [inline src/main/java/org/apache/sling/dynamicinclude/generator/types/JsiGenerator.java:42]: @SBPrakash - Looks like you changed the template, but didn't delete the jquery one, could you delete that as well?

**Review by klcodanr — CHANGES_REQUESTED** [review]

File: src/main/resources/generators/javascript.html (hunk 1 above)
Comment (SBPrakash) [reply src/main/resources/generators/javascript.html:24]: Header - for the purpose just to give an extra hand to developers incase if there is a need to check for header in apache or java..
Timer - page shouldn't be waiting for this request to complete more than 10sec..even 10sec is more, considering lesser bandwidth network may take some time to download resources.

File: src/main/java/org/apache/sling/dynamicinclude/generator/types/JsiGenerator.java (hunk 2 above)
Comment (SBPrakash) [reply src/main/java/org/apache/sling/dynamicinclude/generator/types/JsiGenerator.java:42]: It's removed now.

**Pull request merged**